packages/gatsby/src/utils/__tests__/sample-site-for-experiment.ts

```diff
@@ -1,11 +1,22 @@
 import sampleSiteForExperiment from "../sample-site-for-experiment"
 
+jest.mock(`gatsby-telemetry/lib/repository-id`)
+import {
+  getRepositoryId,
+  IRepositoryId,
+} from "gatsby-telemetry/lib/repository-id"
+;((getRepositoryId as unknown) as jest.Mock<IRepositoryId>).mockReturnValue({
+  repositoryId: `mockId`,
+})
```

Comment:
Import should go first. - jest is smart enough to hoist mocks
```suggestion
import {
  getRepositoryId,
  IRepositoryId,
} from "gatsby-telemetry/lib/repository-id"
;((getRepositoryId as unknown) as jest.Mock<IRepositoryId>).mockReturnValue({
  repositoryId: `mockId`,
})

jest.mock(`gatsby-telemetry/lib/repository-id`)
```